src/services/mouse.js

```diff
@@ -75,16 +90,27 @@ Controller.open(function(_) {
 });
 
 Controller.open(function(_) {
+  /**
+   * 
+   */
   _.seek = function(target, pageX, pageY) {
-    var cursor = this.notify('select').cursor;
+    var cursor = this.cursor;
+
+    // Notify all listeners bound to the Controller that 
+    // a selection has started
+    this.notify('select')
```

Comment:
Again I am not sure we should be making such changes

Reply:
This was more for code navigation purposes. `this.notify` returns `this` when called (probably for chaining). The existing code was ambiguous combining a method call and an assignment into one line.

I think this is spill over from when I was still studying the code. It would be a shame to lose the signposts for anyone else in the future trying to learn this.

Reply:
agreed